packages/flutter_tools/lib/src/commands/build_linux.dart

```diff
@@ -17,6 +17,17 @@ import 'build.dart';
 class BuildLinuxCommand extends BuildSubCommand {
   BuildLinuxCommand({ bool verboseHelp = false }) {
     addCommonDesktopBuildOptions(verboseHelp: verboseHelp);
+    argParser.addOption('target-platform',
```

Comment:
Until we get the pre-cached artifacts downloaded, this command should be updated so that it checks if a local engine is being used with arm64, and throws otherwise. Otherwise the tool will likely crash when it cannot find the binaries to build for these.

(See https://github.com/flutter/flutter/blob/396cd247708f91342070bf292d2cc34315a06c2e/packages/flutter_tools/lib/src/artifacts.dart#L219 )

Reply:
Alternately we could land the engine side (building, uploading) first; that's how we did all of the desktop bringup.

Reply:
Either approach is fine with me